autoscheduler/frontend/src/types/CourseCardOptions.ts

```diff
@@ -27,6 +27,17 @@ export const SortTypeLabels = new Map<SortType, string>([
   [SortType.INSTRUCTOR, 'Instructor'],
   [SortType.OPEN_SEATS, 'Open Seats'],
   [SortType.HONORS, 'Honors'],
+  [SortType.INSTRUCTIONAL_METHOD, 'Instructional Method'],
+]);
+// so that we can reset to defaults on sort type change
+export const SortTypeDefaultIsDescending = new Map<SortType, boolean>([
```

Comment:
I think this should be renamed, as it ending in `IsDescending` makes it sound like a boolean determining whether "SortTypeDefault" is descending or not lol. Maybe `SortTypeDirectionMap` or something (still not a good name but slightly better imo)

Reply:
hows `DefaultSortTypeDirections`

Reply:
Perfect!